src/_nebari/stages/infrastructure/template/local/outputs.tf

```diff
@@ -2,6 +2,7 @@ output "kubernetes_credentials" {
   description = "Parameters needed to connect to kubernetes cluster locally"
   sensitive   = true
   value = {
+    config_path            = var.kubeconfig_filename
```

Comment:
Workaround for a [kubernetes backend provider bug](https://github.com/hashicorp/terraform/issues/31275)